manage.py

```diff
@@ -118,11 +118,11 @@ def pull_prod_data():
     command = "scripts/get_data.sh %s %s" % (prod_db, out_file)
     subprocess.call(shlex.split(command))
 
-    db.session.execute("DELETE FROM association;")
-    db.session.execute("DELETE FROM parent_association;")
-    db.session.execute("DELETE FROM categorisation_value;")
+    db.session.execute("DELETE FROM ethnicity_in_classification;")
+    db.session.execute("DELETE FROM parent_ethnicity_in_classification;")
+    db.session.execute("DELETE FROM ethnicity;")
     db.session.execute("DELETE FROM dimension_categorisation;")
```

Comment:
Should this be updated to `dimension_classification`?

Reply:
@samuelhwilliams we figured that we could not-bother to rename this table, in anticipation of either moving these columns onto the `dimension` table instead, or calculating these values dynamically from the chart and table tables.

Reply:
Cool 👍